cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,50 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
```

Comment:
Class names should be in PascalCase. Rename 'settingsPageObject' to 'SettingsPageObject'.

Reply:
done